introduction.md

```diff
@@ -1,2 +1,5 @@
 Hi! I'm Felipe and I'm new in Dojo!
-I prefer javascript path :)
\ No newline at end of file
+I prefer javascript path :)
+
+* ColdFusion
+* ArangoDB
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?